livedata/src/main/java/io/getstream/chat/android/livedata/extensions/ClientExtensions.kt

```diff
@@ -85,7 +85,7 @@ fun ChatError.isPermanent(): Boolean {
     var isPermanent = true
     if (this is ChatNetworkError) {
         val networkError: ChatNetworkError = this
-        if (networkError.statusCode == 429) {
+        if (networkError.statusCode == 429 || networkError.statusCode == -1) {
```

Comment:
Could we extract those magic number?
BTW, the comment that precedes the code is not up-to-date.
This kind of comments should be removed, they aren't mantained with the new changes are done on the code and could cause confusion.
Instead of that, we need to write some logic that explain the process.
Instead of check with some magic number, they could be into a constant that indicate which status code are permanent error, and check them